plugins/gpio/GPIODriver.h

```diff
@@ -49,6 +49,11 @@ class GPIODriver : private ola::thread::Thread {
      */
     std::vector<uint16_t> gpio_pins;
 
+    /**
+     * @brief A list of gpio_pins which use inverted logic.
+     */
+    std::vector<uint16_t> gpio_inverted_pins;
```

Comment:
Can we make the variable name match the config name please, just to simplify things.